src/app/pages/send/ordinal-inscription/hooks/use-generate-ordinal-tx.ts

```diff
@@ -28,8 +28,8 @@ export function useGenerateSignedOrdinalTx(inscriptionUtxo: TaprootUtxo | Native
   }
 
   function formTrOrdinalTx(values: OrdinalSendFormValues) {
-    const inscriptionInput = inscriptionUtxo as TaprootUtxo;
-    const trSigner = createTapRootSigner?.(inscriptionInput.addressIndex);
+    const inscriptionInput = trInput;
```

Comment:
I'd lengthen `trInput` / `trSigner` to just say `taproot` and be easier to read

```suggestion
    const inscriptionInput = taprootInput;
```

Reply:
Agree cc/ @alter-eggo when off parental leave!